ledger/internal/eval.go

```diff
@@ -59,14 +55,8 @@ type creatable struct {
 	ctype  basics.CreatableType
 }
 
-// FoundAddress is a wrapper for an address and a boolean.
-type FoundAddress struct {
-	Address basics.Address
-	Exists  bool
-}
-
 type roundCowBase struct {
-	l ledgerForCowBase
+	l ledgercore.LedgerForCowBase
```

Comment:
define `LedgerForCowBase` right in `internal` and re-export in for debugging with type aliasing (see below)